indexer-app/src/cmr/indexer/data/concepts/variable.clj

```diff
@@ -5,12 +5,17 @@
    [cmr.common.log :refer (debug info warn error)]
    [cmr.common.util :as util]
    [cmr.indexer.data.elasticsearch :as es]
+   [cmr.indexer.data.concepts.collection.keyword :as k]
```

Comment:
Might be worth moving the keyword namespace out of collection (or create another keyword namespace with a subset of the utility functions useful for both collections and variables) since we now are using keywords in variables too.